File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        for i in range(len(nums)):
            j=i+1
            while(nums[i]==0 and j<len(nums)):
                if(nums[j]!=0):
                    temp=nums[j]
                    nums[j]=nums[i]
                    nums[i]=temp
                else:
                    j+=1
        

File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        for(int i=0;i<nums.length;i++){
            int j=i+1;
            while(nums[i]==0 && j<nums.length){
                if(nums[j]!=0){
                    int temp=nums[j];
                    nums[j]=nums[i];
                    nums[i]=temp;
                } else{
                    j++;
                }
            }
        }
    }
}
